models/models.js

```diff
@@ -44,19 +44,40 @@ exports.fetchArticleById = (id) => {
     });
 };
 
-//5
-exports.fetchArticles = () => {
-  const articlesQuery = `
-    SELECT a.article_id, a.title, a.author, a.topic, a.created_at, a.votes, a.article_img_url, COUNT(c.comment_id) AS comment_count 
-FROM articles AS a 
-LEFT JOIN comments AS c 
-ON c.article_id = a.article_id
-GROUP BY a.article_id
-ORDER BY a.created_at DESC; 
-`;
+//5 and 11 sortby
+exports.fetchArticles = (topic) => {
+  const validTopics = {
+    mitch: 'mitch',
+    cats: 'cats'
+  };
```

Comment:
What if you expand the API in the future to allow clients to add their own articles? Hardcoding it here is not the best way to keep track of the valid articles

Reply:
You could use a topics db query to get the valid topics

Reply:
added query to look for available topics